web_timeline/static/src/css/web_timeline.css

```diff
@@ -0,0 +1,42 @@
+/* Button style */
+.openerp .oe_view_manager .oe_view_manager_switch .oe_vm_switch_timeline:after {
+    content: "N";
+}
+.timeline-navigation-zoom-in .ui-icon{
+  background: none !important;
+}
+
+.timeline-navigation-zoom-out .ui-icon{
+  background: none !important;
+}
+.timeline-navigation-move-left .ui-icon{
+  background: none !important;
+}
+.timeline-navigation-move-right .ui-icon{
+  background: none !important;
+}
+/*.vis.timeline .timeaxis .grid.odd {
+  background: #f5f5f5;
+} */
+
+/* gray background in weekends, white text color */
+.vis.timeline .timeaxis .grid.saturday,
+.vis.timeline .timeaxis .grid.sunday {
```

Comment:
Given we are on v9 now, this file would be much simplified if you use Less or Sass. Not a requirement though.